samples/batch.js

```diff
@@ -40,21 +45,17 @@ function createQueryPartitions(instanceId, databaseId, identifier, projectId) {
 
   const query = 'SELECT * FROM Singers';
 
-  transaction
-    .createQueryPartitions(query)
-    .then(data => {
-      const partitions = data[0];
-      console.log(
-        `Successfully created ${partitions.length} query partitions.`
-      );
-    })
-    .catch(err => {
-      console.error('ERROR:', err);
-    });
+  try {
+    const data = await transaction.createQueryPartitions(query);
```

Comment:
Here (and throughout) - it's a lot easier to just destructure the array during assignment:

```js
const [partitions] = await transaction.createQueryPartitions(query);
```

Then you can drop the next line all together :)